tests/algorithms/test_algorithms.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+import pydp as dp
+
+
+def test_algorithms_BoundedMean():
```

Comment:
This function needs to be shifted to `test_bounded_mean.py`

Reply:
hey @leclair-7, I meant moving it to the different file called `test_bounded_mean.py`

Reply:
Oh gotcha

Reply:
All good now